naoqi_navigation/python/nodes/loading_map.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# Copyright (C) 2017 SoftBank Robotics Europe
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#import ROS dependencies
+import rospy
+
+#import NAO dependencies
+from naoqi_driver.naoqi_node import NaoqiNode
+
+class Exploration(NaoqiNode):
```

Comment:
This class has the same name as the one in exploration.py, should this be called differently ? (MapLoader maybe?)